dts/arm/st/f7/stm32f7.dtsi

```diff
@@ -679,6 +679,14 @@
 			status = "disabled";
 			label = "SDMMC_1";
 		};
+
+		sdmmc2: sdmmc@40011c00 {
```

Comment:
SDMMC2 is not available for stm32f74x/stm32f75x, so please move node definition in stm32f72x/stm32f76x

Reply:
Is there a table to quickly check the features per SoC ?

Reply:
> Is there a table to quickly check the features per SoC ?

I'm usually comparing CMSIS files (https://github.com/zephyrproject-rtos/hal_stm32/tree/master/stm32cube/stm32f7xx/soc)